### DISCUSSION TITLE
Impostazione valutazione dal registro per lo scrutinio

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#246] [Idee e proposte]
Buonasera ,per facilitare a tutto il personale docente l'inserimento delle proposte di voto alunno per alunno , sarebbe possibile effettuare un'importazione dei voti dal registro elettronico che scaturisce dalla media generale dei voti in modo che il docente possa decidere una volta cliccato su importa valutazioni di cambiare il voto che ritiene più opportuno?Grazie , ho visto che nel registro Argo didup funziona così , e tutto il personale docente ,compreso il dirigente ha molto apprezzato questo sistema che velocizza i tempi mando il link che riporta al video :
https://www.youtube.com/watch?v=SaUCkVs2VCs

**Comment by trinko** [comment]
Questa non è una segnalazione di errore, ma una richiesta: la sposto nell'apposito forum di discussione.
